udata_gouvfr/theme/gouvfr/templates/organization/producer-panel.html

```diff
@@ -19,7 +19,7 @@ <h2>
                     title="{{ _('Contact the producer') }}">{{ _('Contact the producer') }}</a></li>
             {# <li><a class="nav-link text-decoration-none text-blue-500"
                     href="{{ url_for('api.organization_followers', id=organization.id|string) }}"
-                    title="">{{ _('@@Suivre le producteur') }}</a></li> #}
+                    title="">{{ _('Follow the producer') }}</a></li> #}
```

Comment:
```suggestion
                    title="">{{ _('Add the producer as favorite') }}</a></li> #}
```

Reply:
Favorite or Favourite, you got to pick :p

Reply:
good catch 🎣